src/MainDemo.Wpf/ColorZones.xaml

```diff
@@ -38,6 +38,7 @@
               <ListBoxItem Content="Hello World" />
               <ListBoxItem Content="Nice Popup" />
               <ListBoxItem Content="Goodbye" />
+              <ListBoxItem Content="Version" />
```

Comment:
Having something app-specific on a page which showcases a Material Design component is debatable, but I don't mind that much